numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -10,13 +10,29 @@
 
 """
 from __future__ import division, absolute_import, print_function
+import textwrap
 
 docdict = {}
 
 def get(name):
     return docdict.get(name)
 
+# common parameter text to all ufuncs
+_params_text = textwrap.dedent("""
+    out : ndarray, optional
+        Alternate array object(s) in which to put the result; if provided, it
+        must have a same shape that the inputs broadcast to.
+    where : array_like, optional
+        Values of True indicate to calculate the ufunc at that position, values
```

Comment:
As a bonus, this now means that `where` is documented everywhere.

We could add the others here, but the explanations are more complicated